src/hooks/useHashLocation.ts

```diff
@@ -0,0 +1,25 @@
+import { BaseLocationHook } from 'wouter'
+import { useEffect, useState } from 'preact/hooks'
+
+// returns the current hash location in a normalized form
+const currentLocation = () => {
+  return window.location.hash.replace(/^#/, '') || '/'
+}
+
+const navigate = (to: string) => (window.location.hash = to)
```

Comment:
Should we use this function everywhere where we use button with location? Please check if the buttons work

Reply:
No need to do that, cause it's already works as it should